test/ctia/http/generative/fulltext_search_test.clj

```diff
@@ -106,110 +76,138 @@
+       :query-params     {:query_mode    "query_string"
+                          :query         "\"title of test incident\" AND \"Log Review\""
                           :search_fields ["title" "discovery_method"]}
-       :bundle-gen       bundle
+       :bulk-gen         bulk
        :check            (fn [_ _ _ res]
                            (is (= 1 (-> res :parsed-body count)))
                            (is (get-fields res)))}])
 
    [{:test-description "multi_match - looking for the same value in different fields in multiple records"
-     :query-params     {:query_mode      "multi_match"
-                        :query           "bibendum"
+     :query-params     {:query_mode    "multi_match"
+                        :query         "bibendum"
                         :search_fields ["assignees" "title"]}
-     :bundle-gen       (gen/fmap
-                        (fn [bundle]
+     :bulk-gen         (gen/fmap
+                        (fn [examples]
                           (update
-                           bundle :incidents
+                           examples :incidents
                            (fn [incidents]
                              (apply
                               utils/update-items incidents
                               (repeat 3 ;; update first 3, leave the rest unchanged
                                       #(assoc % :title "Etiam vel neque bibendum dignissim"
                                               :assignees ["bibendum"]))))))
-                        (bundle-gen-for :incidents))
+                        (bulk-gen-for :incidents))
      :check            (fn [_ _ _ res]
                          (let [matching (->> res
                                              :parsed-body
                                              (filter #(and (-> % :assignees (= ["bibendum"]))
                                                            (-> % :title (= "Etiam vel neque bibendum dignissim")))))]
-                           (is (= 3 (count matching)))))}]))
+                           (is (= 3 (count matching)))))}]
 
-(defn test-search-case
-  [app
-   {:keys [query-params
-           bundle-gen
-           check
-           test-description] :as test-case}]
-  (let [services (app/service-graph app)
-        bundle   (gen/generate bundle-gen)
-        ent-keys (->> bundle
-                      keys
-                      (filter (partial contains? bundle-entity-field-names)))]
-    (bundle/import-bundle
-     bundle
-     nil    ;; external-key-prefixes
-     login
-     services)
+   [{:test-description "searching for non-existing fields, returns errors"
+     :query-params     {:query         "*"
+                        :search_fields ["donec_retium_posuere_tellus"
+                                        "proin_neque_massa"]}
+     :bulk-gen         (bulk-gen-for :incidents)
+     :check            (fn [_ _ _ res]
+                         (is (-> res :parsed-body (contains? :errors))))}
```

Comment:
```suggestion
                         (is (= 400 (-> res :parsed-body :status))))}
```

Reply:
indeed it's already tested above:
"https://github.com/threatgrid/ctia/pull/1127/files#diff-7c52458f6c78a79afff4de6bfb3123a41d720dad0bfb7eb0ef82ea67d0421a2bR102"